fhir-server/src/main/java/com/ibm/fhir/server/util/RestAuditLogger.java

```diff
@@ -498,6 +491,8 @@ private static AuditLogEntry populateAuditLogEntry(AuditLogEntry entry, HttpServ
                             .append("/")
                             .append(request.getRemoteHost()).toString());
         entry.setContext(new Context());
+
+        // TODO: Unify with the FHIRRequestContext
```

Comment:
the one update I think you missed :-)

Reply:
Thanks, it's a online change, so easy to get in.